### PR TITLE
Miles: Timestamp bug fix

### PR DESCRIPTION
**Pull request by milesflo**
The build's problem lied in the built-in date parser broken, time_ago_in_words(). It was passed quote.created_at , the timestamp of posting. Unfortunately, not all entries in the quotes datatable had input for the created_at column. Because time_ago_in_words() was passed an invalid datatype (null), the app crashed. To fix this, I altered the data parser to include an else statement of Time.now, 

> time_ago_in_words(quote.created_at || Time.now)

such that it display "less than a minute ago" when no time stamp is present.

### CODE REVIEW

**Comment by tebba-von-mathenstein** [comment]
Hey Miles,

Nice work here -- It looks like you've prevented an error from displaying, but it sounds like we still have some bad data in the database. The `created_at` field should _never_ be nil, can you fix the production.sql file and give any quotes without a created_at timestap todays timestamp. 

Additionally can you add a constraint to the quotes model, such that created_at can never be nil? 

Thanks a lot.

File: app/views/quotes/index.html.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+      <%= time_ago_in_words(quote.created_at || Time.now) %> ago
```
Comment (tebba-von-mathenstein) [inline app/views/quotes/index.html.erb]: This feels more like a workaround -- lets revert this change and make sure the data can never have a nil timestamp instead. 

We _ought to_ experience an error if the timestamp is nil, so I want to leave this code in a way that causes an error if thats the case.

Comment (milesflo) [inline app/views/quotes/index.html.erb]: Understood

**Comment by milesflo** [comment]
Removed the previous workaround and changed the problematic entries in the database to always be created at the current date and time.

**Comment by tebba-von-mathenstein** [comment]
Thanks Miles, this LGTM. I'll handle the merge to master, and close this PR. :)

File: Gemfile.lock [hunk 2]
```diff
@@ -54,7 +54,7 @@ GEM
     jquery-rails (3.1.2)
       railties (>= 3.0, < 5.0)
       thor (>= 0.14, < 2.0)
-    json (1.8.1)
+    json (1.8.3)
```
Comment (tebba-von-mathenstein) [inline Gemfile.lock:57]: hey why did you bump this version? I don't think we need 1.8.3.

Comment (tebba-von-mathenstein) [inline Gemfile.lock:57]: miles says: Yo, actually 1.8.1 contains a known compatibility issue with ruby 2.2.x and we need to update to continue using this version of ruby!

Comment (tebba-von-mathenstein) [inline Gemfile.lock:57]: Oh, thanks! Great then lets keep it!

File: spec/features/timestamp_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+require 'capybara/rails'
```
Comment (tebba-von-mathenstein) [inline spec/features/timestamp_spec.rb:2]: Hey waht are these used for again?
